package.json

```diff
@@ -17,7 +17,7 @@
         "lint:styles": "stylelint 'src/**/*.s(a|c)ss'",
         "test": "jest --config scripts/test/jest.config.js",
         "test:coverage": "jest --config scripts/test/jest.config.js --coverage",
-        "prepublishOnly": "npm test && npm run lint:ci && npm run lint:styles && npm run build:prod",
+        "prepublishOnly": "sh scripts/check-ownership/check-ownership.sh && npm test && npm run lint:ci && npm run lint:styles && npm run build:prod",
```

Comment:
Can we move `sh scripts/check-ownership/check-ownership.sh` to separate command?
Profit: 
- we can run check ownership without the whole build process;
- `prepublishOnly` command will be shorter;
What do you think?

Reply:
good idea